app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/general_fee/AddGeneralFee.kt

```diff
@@ -14,7 +14,17 @@ class AddGeneralFee : OsmFilterQuestType<Boolean>(), AndroidQuest {
     override val elementFilter = """
         nodes, ways with
          (
-           tourism ~ museum|gallery|caravan_site
+           (""" +
+        // Bicycle charging station are amenity=charging_station with bicycle=yes/designated.
+        // We further exclude dual use charging stations
```

Comment:
```suggestion
        // We further exclude dual use charging stations as they are expected to be paid, with extremely rare exceptions
```

is that reason for excluding them? or is it some other one?

Reply:
I think the main issue is that the fee status might very likely differ for e-bicycles and e-cars (former often being free of charge, and later often paid) and that cannot be answered with simple `fee=*` tag anymore (one would have to resort to little used `fee:bicycle=*`+`fee:motorcar=*` combinations, or `fee=yes` + `fee:conditional=no @ bicycle` etc.)

So this is long hanging fruit. To also solve dual-use charging stations, it would require:
- more complex tagging, which is not yet properly documented in the wiki nor in widespread use, so probably community discussion first too
- separate quest which would ask something like _"Select all of the following vehicle types which can charge here for free"_ (and then dynamically generate the list depending on existing `truck|motorcar|bicycle|... = yes|designated` tags)

Reply:
edited it then

my motivation is to be aware of "why" if someone arrives and proposes to expand quest to all charging stations